crabserver/templates/deployment.yaml

```diff
@@ -0,0 +1,180 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "crabserver.fullname" . }}
+  labels:
+    {{- include "crabserver.labels" . | nindent 4 }}
+  namespace: crab
+spec:
+  {{- if not .Values.autoscaling.enabled }}
+  {{- if or .Values.environment.prod .Values.environment.preprod}}
+  replicas: 5
+  {{- end}}
```

Comment:
Put else here with if condition. By default replica is 1 from values.yaml file, for prod and preprod it is 5.